tox.ini

```diff
@@ -16,6 +16,11 @@ commands =
 [testenv:test]
 passenv =
   BUILD_IN_DOCKER
+  DOCKER_ENV_VARS
+  DOCKER_MAKE_ARGS
+  GITHUB_REPOSITORY
+  GITHUB_RUN_ID
+  GITHUB_SERVER_URL
```

Comment:
Why are these added?

Reply:
Oops, those belong to #173. Must be a leftover from when I took this out of #174.

Reply:
Moved to #173